src/app/components/filter/filter-panel.jsx

```diff
@@ -225,17 +247,18 @@ function ChplFilterPanel() {
               </List>
             </div>
           </div>
-          <div>
+          <div className={classes.subfilters}>
             { activeCategory?.values.length > 0 && (
               <List
                 dense
                 subheader={(
                   <ListSubheader
-                    disableSticky
                     component="div"
                     id="filter-panel-secondary-subheader"
-                  >
-                    <div className={classes.filterSubHeaderContainer}>
+                    className={classes.clearResetContainer}
+                    disableGutters
+                    >
+                    <div>
```

Comment:
Why does this empty div exist now? I think it used to be there to hold the class, but if the class moved up a level do we still need the div?